pkgs/tools/graphics/ldgallery/viewer/default.nix

```diff
@@ -24,6 +24,7 @@ let
   nodePkg = nodePackages.package.override {
     src = "${sourcePkg}/viewer";
     postInstall = "npm run build";
+    buildInputs = optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.CoreServices ];
```

Comment:
```suggestion
    buildInputs = lib.optionals stdenv.isDarwin [ CoreServices ];
```

Reply:
See https://github.com/NixOS/nixpkgs/pull/123226#discussion_r633232262

Reply:
darwin shouldn't be used as an input.

Reply:
Putting `CoreServices` directly as an input in the parameter attribute set results in the following error:

```
error: anonymous function at /home/kea/src/nixpkgs/pkgs/tools/graphics/ldgallery/viewer/default.nix:1:1 called without required argument 'CoreServices', at /home/kea/src/nixpkgs/lib/customisation.nix:69:16
```

I also tried putting `apple_sdk` as a parameter, but this results in the same error.

Reply:
This is now fixed and amended. I just needed to inherit `CoreServices` properly.